src/v2/Components/Pagination/index.tsx

```diff
@@ -63,17 +69,24 @@ export const Pagination: React.FC<Props> = ({
   }
 
   return (
-    <ScrollIntoView selector={scrollTo}>
-      <Media at="xs">
-        <SmallPagination {...paginationProps} />
-      </Media>
-      <Media greaterThan="xs">
-        <Box>
-          <Separator mb={3} pr={2} />
-          <LargePagination {...paginationProps} />
-        </Box>
-      </Media>
-    </ScrollIntoView>
+    <>
+      {tokens.version === "v2" && (
+        <>
+          <Media at="xs">
+            <SmallPagination {...paginationProps} />
+          </Media>
+
+          <Media greaterThan="xs">
+            <Separator mb={3} />
+            <PaginationBase {...paginationProps} />
+          </Media>
+        </>
+      )}
+
+      {tokens.version === "v3" && (
+        <PaginationBase mt={6} {...paginationProps} />
```

Comment:
This might be an update for Palette repo, but we shouldn't need to think about `ThingBase` when using `<Pagination>`, a bit too confusing.